src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -156,10 +164,7 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
         })
       })
     } else {
-      // Filter out duplicate accessors coming from existing ARs
-      const uniqueAccessorChanges = accessorChanges.filter((user, i, arr) =>
-        arr.findIndex(accArr => (accArr.userId === user.userId)) === i)
-      uniqueAccessorChanges.forEach(item => {
+      accessorChanges.forEach(item => {
         ids.push(item.userId)
```

Comment:
They were being filtered out for the display, but submitting the existing data access request still failed (because the duplicates were still in the accessorChanges)

Reply:
Yes, this is better! Thank you.